cypress/e2e/column-text-link.cy.js

```diff
@@ -0,0 +1,79 @@
+let localUser
+
+describe('Test column text-link', () => {
+
+	before(function() {
+		cy.createRandomUser().then(user => {
+			localUser = user
+		})
+	})
+
+	beforeEach(function() {
+		cy.login(localUser)
+		cy.visit('apps/tables')
+	})
+
+	it('Create', () => {
+		cy.contains('.app-menu-entry--label', 'Tables').click()
+		cy.contains('button', 'Create new table').click()
+		cy.get('.tile').contains('Custom table').click({ force: true })
+		cy.get('.modal__content input[type="text"]').clear().type('Test text-link')
+		cy.contains('button', 'Create table').click()
+
+		cy.contains('h1', 'Test text-link').should('be.visible')
+	})
+
+	it('Create 3 text-link columns', () => {
+		// first column
+		cy.get('.app-navigation-entry-link').contains('Test text-link').click({ force: true })
+		cy.get('.button-vue__text').contains('Create column').click({ force: true })
+		cy.get('.modal-container').get('input[placeholder*="Enter a column title"]').clear().type('Test plain url')
+		cy.get('.columnTypeSelection .vs__open-indicator').click({ force: true })
+		cy.get('.multiSelectOptionLabel').contains('Link').click({ force: true })
+		// deactivate unwanted provider
+		cy.get('.typeSelection span label').contains('Files').click()
+		cy.get('.typeSelection span label').contains('Contacts').click()
+		cy.get('.modal-container button').contains('Save').click()
+
+		cy.wait(10).get('.toastify.toast-success').should('be.visible')
```

Comment:
Maybe intercept the request and wait for that instead of the seconds ;)

Reply:
https://docs.cypress.io/api/commands/wait#Wait-for-a-specific-request-to-respond

Basically:
- intercept the request, give it a name
- perform the action like clicking save
- wait for the intercepted alias name